### PR TITLE
Add gender option

### PR DESCRIPTION
**Pull request by Dhroov7**

### CODE REVIEW

File: views/signup.hbs [hunk 1]
```diff
@@ -11,6 +11,11 @@
     <input class="form-control fontAwesome inputvalidation" name="username" placeholder="&#xf007; Username" type="text" required> <br>
     <input class="form-control fontAwesome inputvalidation" name="firstname" placeholder="&#xf061; Firstname" type="text" required> <br>
     <input class="form-control fontAwesome inputvalidation" name="lastname" placeholder="&#xf061; Lastname" type="text" required> <br>
+    <label>Gender:</label>
+    <select class="form-control" name="gender" required>
+        <option>Male</option>
```
Comment (championswimmer) [inline views/signup.hbs]: We need "Do not disclose" option too

**Comment by Dhroov7** [comment]
@championswimmer Resolved all the conflicts and added migration too...Please review.

File: src/db/models.js [hunk 2]
```diff
@@ -32,6 +32,7 @@ const User = db.define('user', {
     username: {type: Sequelize.DataTypes.STRING, unique: true, allowNull: false},
     firstname: Sequelize.DataTypes.STRING,
     lastname: Sequelize.DataTypes.STRING,
+    gender: {type:Sequelize.DataTypes.STRING,defaultValue:'Don\'t want to disclose'},
```
Comment (championswimmer) [inline src/db/models.js]: ENUM('MALE', 'FEMALE', 'UNDISCLOSED')

**Comment by Dhroov7** [comment]
@championswimmer Done with the changes.

**Comment by Dhroov7** [comment]
@championswimmer Resolved conflicts.

File: src/db/models.js [hunk 3]
```diff
@@ -32,6 +32,7 @@ const User = db.define('user', {
     username: {type: Sequelize.DataTypes.STRING, unique: true, allowNull: false},
     firstname: Sequelize.DataTypes.STRING,
     lastname: Sequelize.DataTypes.STRING,
+    gender: {type:Sequelize.DataTypes.ENUM('MALE','FEMALE','UNDISCLOSED')},
```
Comment (championswimmer) [inline src/db/models.js]: need to set default UNDISCLOSED as all the existing students

**Pull request merged**